docs/index.rst

```diff
@@ -96,7 +96,7 @@ ABM features users have shared that you may want to use in your model
 
    Mesa Overview <overview>
    tutorials/intro_tutorial
```

Comment:
Is there a reason `adv_tutorial` is `.ipynb` but `intro_tutorial` is not?

Reply:
I was adding minimal amount of changes to the baseline v1.2, because the PR modified only the intro tutorial.

Reply:
That's fair